src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -180,10 +181,22 @@ const convertFrameworkSummary = (framework) => {
 const convertTaskDetail = async (taskStatus, ports, logPathPrefix) => {
   // get container ports
   const containerPorts = {};
-  if (ports) {
+  if (ports && taskStatus.attemptStatus.podUID) {
     const randomPorts = JSON.parse(ports);
-    for (let port of Object.keys(randomPorts)) {
-      containerPorts[port] = randomPorts[port].start + taskStatus.index * randomPorts[port].count;
+    if (randomPorts.ports) {
+      for (let port of Object.keys(randomPorts.ports)) {
+        const portNums = [...Array(randomPorts.ports[port].count).keys()].map((index) => {
+          const rawString = taskStatus.attemptStatus.podUID + port + index;
+          return parseInt(crypto.createHash('md5').update(rawString)
```

Comment:
Add landmark, so it would be const rawString = [ + taskStatus.attemptStatus.podUID +] + [ + port +] + [ + index +];

Reply:
Sorry, don't get your point.... With/Without landmark seems does't impact anything

Reply:
If no mark, port 12 + index 3 map to the same rawString 123 as port 1 + index 23.
But if use mark, the result rawString will be different, such as [12][3] and [1][23].
Let's keep the source info as more as possible, so the final port will be more rand,.

Reply:
Fixed